server/supernova/audio_backend/jack_backend.hpp

```diff
@@ -270,7 +271,11 @@ class jack_backend:
     {
         std::atomic_thread_fence(std::memory_order_acquire);
         jack_backend * self = static_cast<jack_backend*>(arg);
+#ifdef _WIN32
+        if (jack_client_thread_id(self->client) == GetCurrentThread())
```

Comment:
latest windows binaries are built using mingw, which uses pthreads, so this check will fail